flake.nix

```diff
@@ -67,6 +67,10 @@
               excludes = [ ".*spago-packages.nix$" ];
             };
             inherit purs-tidy-hook;
+            prettier = {
+              enable = true;
+              types_or = [ "javascript" "css" "html" ];
+            };
```

Comment:
is this applied to the old JS files or just new commits? we might want to do a separate PR in which we apply prettier to old files.

Reply:
nevermind... just saw the changes

Reply:
Few enough files in this repo that I don't think it's a huge deal.  I'm going to do dedicated formatting PRs for the other repos.